test/models/user_test.rb

```diff
@@ -31,25 +30,25 @@ def setup
     assert_not @user.valid?
   end
 
-  test "email validation should accept valid addresses" do
-    valid_addresses = %w[user@example.com USER@foo.COM A_US-ER@foo.bar.org
-                         first.last@foo.jp alice+bob@baz.cn]
+  test 'email validation should accept valid addresses' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.